app/models/enrollment.rb

```diff
@@ -0,0 +1,11 @@
+class Enrollment < ApplicationRecord
+  belongs_to :student, class_name: "User"
+  belongs_to :course
+
+  STATUS = {not_started: 1, in_progress: 2, completed: 3}.freeze
+  enum completion_status: STATUS
```

Comment:
ditto

Reply:
Apu, actually Sarah apu suggested this: [#2 (comment)](https://github.com/amu-wtag/well_academy/pull/2#discussion_r1839875861) . Should I keep it or change?

Reply:
ditto